DemoScript.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
+
+<suite name="DemoScript" thread-count="10" parallel="tests">
+    <parameter name="browser" value="chrome"/>
+    <parameter name="resolutions" value="1024x768,1496x967,1920x1080"/>
+    <parameter name="url" value="https://smartbear.github.io/visual-testing-example-website/Example1/Original/index.html"/>
+    <test name="DemoScript0">
+        <classes>
+            <class name="DemoScript"/>
+        </classes>
+    </test>
+    <test name="DemoScript1">
```

Comment:
You don't have to write 10 times the `<test>` block since you are launching the same tests. There is a property called `invocationCount` that you can set to 10.